editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -1277,7 +1280,7 @@ void EditorAssetLibrary::disable_community_support() {
 }
 
 void EditorAssetLibrary::_bind_methods() {
-	ClassDB::bind_method("_unhandled_input", &EditorAssetLibrary::_unhandled_input);
+	ClassDB::bind_method("_unhandled_button_input", &EditorAssetLibrary::_unhandled_button_input);
```

Comment:
I think it should remain as _unhandled_key_input, what I meant is that if we at any point need to add button support (like joypad, mouse, etc) we can add an _unhandled_button_input, but we need to wait to see if this is something needed or that makes sense for users at some point.

Reply:
I personally believe it would be good to have shortcuts support mouse buttons, it's a good to have for not much trouble.
But still, I would avoid adding yet another function only this. IMHO it's better to have all those events inside an `_unhandled_key_or_button_input`, so that nodes implementing shortcuts would have only one function to override, not two.

Reply:
Via discussion in IRC it was agreed that handling Mouse Button events is more complicated than it looks at the surface.

For this reason, the API will remain as `_unhandled_key_input` and only support keys at this stage.